src/main/java/com/fasterxml/jackson/databind/node/ArrayNode.java

```diff
@@ -248,7 +248,7 @@ public int size() {
 
     @Override
     public Iterator<JsonNode> elements() {
-        return _children.iterator();
+        return _children.listIterator();
```

Comment:
Could you add a comment wrt references to issue, and version in which change goes in (2.18)?

Reply:
Actually n/m I can do this.